src/impact-dashboard/CurveChartContainer.tsx

```diff
@@ -6,10 +6,13 @@ import { getAllValues, getColView } from "../infection-model/matrixUtils";
 import { seirIndex } from "../infection-model/seir";
 import { MarkColors } from "./ChartArea";
 import CurveChart from "./CurveChart";
-import { useEpidemicModelState } from "./EpidemicModelContext";
+import { EpidemicModelState } from "./EpidemicModelContext";
 
 interface Props {
+  chartHeight?: number;
   markColors: MarkColors;
+  // This isn't actually optional, but disabling it for PR purposes
+  modelData?: EpidemicModelState;
```

Comment:
Passing `modelData` in as a property at the facility level now that it's no longer "global" for the entire page.

Reply:
I think when this shakes out we will still want one `EpidemicModelContext` per facility (and one chart per facility) so in the end you shouldn't have to do this and can revert to the previous data fetching strategy.

Reply:
That's actually great news.  I'll try to wire things up accordingly